mls-rs/src/group/ciphertext_processor/reuse_guard.rs

```diff
@@ -92,23 +92,27 @@ mod tests {
         result: Vec<u8>,
     }
 
+    #[cfg_attr(coverage_nightly, coverage(off))]
     fn generate_reuse_guard_test_cases() -> Vec<TestCase> {
         let provider = test_cipher_suite_provider(TEST_CIPHER_SUITE);
 
         [16, 32]
             .into_iter()
-            .map(|len| {
-                let nonce = provider.random_bytes_vec(len).unwrap();
-                let guard = ReuseGuard::random(&provider).unwrap();
-
-                let result = guard.apply(&nonce);
-
-                TestCase {
-                    nonce,
-                    guard: guard.into(),
-                    result,
-                }
-            })
+            .map(
+                #[cfg_attr(coverage_nightly, coverage(off))]
```

Comment:
Why is it necessary to place that on the lambda definition when the containing function already has `coverage(off)`?

Reply:
It won't ignore it otherwise. Pretty sure this is a nuance of LLVM where those lambdas are internally converted into anonymous functions

Reply:
Thank you for explaining.